_data/simple-icons.json

```diff
@@ -5870,6 +5870,11 @@
             "hex": "003A9B",
             "source": "https://indeed.design/resources"
         },
+        {
+            "title": "IndiGo",
+            "hex": "FFFFFF",
```

Comment:
```suggestion
            "hex": "09009B",
```

Reply:
Happy with the SVG now, but I'd still go with the hex change suggested here. Once that's done - we can get this merged!